tests/common/dependency_helper.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <optional>
+
+#include <winrt/base.h>
+#include <winrt/Windows.Management.Deployment.h>
+#include <winrt/Windows.ApplicationModel.h>
+#include <winrt/Windows.Foundation.Collections.h>

```

Comment:
Looks like these files need to be compiled only once - Is there a possibility to add something similar to pch.h?